### PR TITLE
Fix mixup between `translated` and `translated_local`

### PR DESCRIPTION
**Pull request by AThousandShips**
* Fixes: https://github.com/godotengine/godot-docs/issues/9428

### CODE REVIEW

File: tutorials/math/matrices_and_transforms.rst [hunk 1]
```diff
@@ -222,11 +222,12 @@ to set its ``origin`` vector to (1, 2):
...
 
-There is also a ``translated()`` method, which performs a different
-operation to adding or changing ``origin`` directly. The ``translated()``
+There is also a ``translated_local()`` method, which performs a different
+operation to adding or changing ``origin`` directly. The ``translated_local()``
 method will translate the object *relative to its own rotation*.
 For example, an object rotated 90 degrees clockwise will move to
-the right when ``translated()`` with ``Vector2.UP``.
+the right when ``translated_local()`` with ``Vector2.UP``. To translate
+*relative to the global/parent frame* use ``translated()`` instead.
```
Comment (AThousandShips) [inline tutorials/math/matrices_and_transforms.rst:230]: Took this wording directly from `Transform3D.translated`

**Review by Calinou — APPROVED** [review]
Looks good to me.

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thank you, merged.

**Comment by AThousandShips** [comment]
Thank *you*!

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.2 in https://github.com/godotengine/godot-docs/pull/9648.

**Pull request merged**